audb/core/load.py

```diff
@@ -899,10 +899,28 @@ def load(
         mixdown: apply mono mix-down
         sampling_rate: sampling rate in Hz, one of
             ``8000``, ``16000``, ``22500``, ``44100``, ``48000``
-        tables: include only tables matching the regular expression or
-            provided in the list
-        media: include only media matching the regular expression or
-            provided in the list
+        attachments: load only attachment files
+            for the attachments
+            matching the regular expression
+            or provided in the list.
+            Use ``[]`` to not load any attachment files
+        tables: include only tables
+            matching the regular expression
+            or provided in the list.
+            It will also limit possible media files
+            to those listed in the selected tables.
+            If ``[]`` is provided
+            only misc tables
+            used as labels in a scheme
+            and no media files are loaded
+        media: include only media files
+            matching the regular expression
+            or provided in the list.
+            The entries for non-matching media files
+            will be removed from tables
+            resulting in potentially empty tables.
+            If ``[]`` is provided
+            no media files are loaded
```

Comment:
```suggestion
        media: load only media files
            matching the regular expression
            or provided in the list.
            Excluded media files are 
            automatically removed from the tables.
            This may result in empty tables.
            If set to ``[]``
            no media files are loaded
            and all tables except
            misc tables will be empty
```